### PR TITLE
[:feat] add walletconnect/modal example to lab

### PR DESCRIPTION
**Pull request by svenvoskamp**
# Changes

- feat: add walletconnect/modal example to lab

# Associated Issues

closes https://github.com/WalletConnect/web3modal/issues/1066

### CODE REVIEW

File: laboratory/src/pages/index.tsx [hunk 1]
```diff
@@ -129,6 +139,42 @@ export default function HomePage() {
           </Grid>
         ))}
       </Grid.Container>
+
+      <Container css={{ maxWidth: '940px', margin: '50px auto 0' }}>
+        <Text h3 color="gray">
+          @walletconnect/modal Playgrounds
```
Comment (0xAsimetriq) [inline laboratory/src/pages/index.tsx:145]:
```suggestion
          WalletConnectModal Playgrounds
```

**Review by 0xAsimetriq — COMMENTED** [review]
Can we make this consistent with other examples i.e. sign ones:

- Show session data after connecting / use same modals
- When user connects, show disconnect and sign message buttons
- Add sign message request to test

Also, needs to be updated against v2 branch

**Comment by svenvoskamp** [comment]
This example is now consistent with other sign examples. 

* Session data is visible after connection (using same modal as other examples)
* Possibility to test signing a message 
* Updated to current V2 branch 
* Correct naming 

Could you please review again?

**Comment by 0xAsimetriq** [comment]
> This example is now consistent with other sign examples.
> 
> * Session data is visible after connection (using same modal as other examples)
> * Possibility to test signing a message
> * Updated to current V2 branch
> * Correct naming
> 
> Could you please review again?

One small task, at the moment page doesn't react to me disconnecting from mobile wallet i.e.
1. Open lab on zerion
2. Connect
3. Disconnect from zerion (lab is still connected)

**Review by 0xAsimetriq — APPROVED** [review]

**Review by 0xAsimetriq — APPROVED** [review]

**Pull request merged**